addons/purchase_stock/models/purchase_order.py

```diff
@@ -37,6 +37,25 @@ def _default_picking_type(self):
        help="Red: Late\n\
             Orange: To process today\n\
             Green: On time")
+    suggest_estimated_price = fields.Float(compute='_compute_suggest_estimated_price')
+
+    @api.depends_context("suggest_based_on", "suggest_percent", "suggest_days", "warehouse_id", "hashable_domain")
+    def _compute_suggest_estimated_price(self):
+        ctx = self.env.context
+        for purchase_order in self:
+            estimated_price = 0
+            seller_args = {
+                "partner_id": purchase_order.partner_id,
+                "params": {'order_id': purchase_order}
+            }
+            products = self.env['product.product'].with_context(ctx).search(ctx.get('domain', []))
```

Comment:
maybe add product.type, '=', 'consu' anyway
domain = [('type', '=', 'consu')]
if self.env.context.get('domain'):
    domain = Domain.AND(domain, self.env.context.get('domain'))